### PR TITLE
Functional tests - Add new test 'Check notifications in BO'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add new test 'Check notifications in BO'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | https://github.com/nesrineabdmouleh/testing_pr/actions/runs/3995288460
| Fixed ticket?     | no
| Related PRs       | no
| Sponsor company   | no

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/15_header/08_notifications.ts [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+- Create order by guest in FO
+Scenario:
+- Go to BO and check notifications in Customers and orders tab
+- Click on each notification and check order and customer page
+ */
+describe('BO - Header : Check notifications', async () => {
+  let browserContext: BrowserContext;
+  let page: Page;
+
+  const orderByCustomerData: Order = {
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/15_header/08_notifications.ts]:
```suggestion
  const orderByCustomerData: OrderData = new OrderData({
```

File: tests/UI/campaigns/functional/BO/15_header/08_notifications.ts [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+  const messageOption: string = `${Products.demo_1.name} (Size: ${Products.demo_1.attributes[0].values[0]} `
+    + `- Color: ${Products.demo_1.attributes[1].values[0]})`;
+  const customerData: CustomerData = new CustomerData({password: ''});
+  const addressData: AddressData = new AddressData({country: 'France'});
+  // New order by guest data
+  const orderByGuestData: Order = {
+    customer: customerData,
+    productId: 4,
+    productQuantity: 1,
+    address: addressData,
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/15_header/08_notifications.ts]: I don't see where it is used

**Review by Progi1984 — CHANGES_REQUESTED** [review]
Some feedbacks

File: tests/UI/campaigns/functional/BO/15_header/08_notifications.ts (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/15_header/08_notifications.ts]: Thank you @Progi1984 :)
Done !
Yes I fixed the conflicting files after your modifications !

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by nicosomb** [comment]
Thank you @nesrineabdmouleh !

**Pull request merged**